src/graph.cpp

```diff
@@ -217,7 +221,7 @@ std::vector<std::set<node_id_t>> Graph::boruvka_emulation() {
   for (const auto& it : temp) retval.push_back(it.second);
 
   cc_alg_end = std::chrono::steady_clock::now();
-  printf("CC done\n");
+  printf("CC done in %d rounds\n", num_rounds);
```

Comment:
Probably not necessary

Reply:
I used this for some experiments, but I'm not too attached to it if you don't like it

Reply:
Yeah lets get rid of it. We can always add it back if we want to. Also, the cc done printf isn't necessary either.